layouts/partials/episode-module.html

```diff
@@ -1,22 +1,22 @@
-{{ $.Scratch.Set "imagePath" (printf "episodes/%s/artwork.png" (string .Params.number)) }}
-{{ $.Scratch.Set "imagePathHighRes" (replace (.Scratch.Get "imagePath") ".png" "-high-res.png") }}
-{{ $.Scratch.Set "itunesTracking" (printf "episode%s" (string .Params.number)) }}
-{{ $.Scratch.Set "rpTracking" (printf "episode%s-modal" (string .Params.number)) }}
-{{ if in .Params.number "B" }}
+{{ if in .Params.seasonepisode "B" }}
```

Comment:
Maybe add comments here and in the `else if` below to make it obvious what conditions are being checked, e.g. bonus episodes and trailers.